lib/dl_api_lib/dl_api_lib/app/control_api/resources/info.py

```diff
@@ -130,12 +130,12 @@ def get(self, conn_type: str, form_mode: str) -> dict:
         try:
             ct = ConnectionType(conn_type)
         except ValueError:
-            raise UnsupportedForEntityType(f"Unknown connector type: {conn_type}")
+            raise UnsupportedForEntityType(f"Unknown connector type: {conn_type}") from None
```

Comment:
let's myte this one (not line but the whole check)

Reply:
Sure
Can we keep what I've added?

Reply:
why not, just let's mute it for future generations to come